CMakeLists.txt

```diff
@@ -30,7 +34,9 @@ include(CMakePrintHelpers)
 include(F2Py)
 find_package(pybind11 CONFIG)
 
-find_package(Python REQUIRED COMPONENTS Interpreter Development)
+
+message(STATUS "VIRTUAL_ENV = $ENV{VIRTUAL_ENV}")
```

Comment:
This message looks like it was useful during debugging but should be removed. find_package should find the python in the virtual environment if you are in a virtual environment, otherwise that would be a bug in find_package(Python ...)

Reply:
I would say that it is rather a bug of `cibuildwheel`. When the virtual environment is activated, then VIRTUAL_ENV is set. `cibuildwheel` doesn't do that. That is the problem in this case, and the only workaround is to set it by hand (or activate the virtual environment somehow).